### DISCUSSION TITLE
Ayaha Hideri的形象设计和来源是什么？

### DISCUSSION THREAD

**Discussion by specked2527** [apernet/hysteria#1151] [Q&A] [answered]
没有看到说明捏
![23](https://github.com/user-attachments/assets/25bd8858-7ad1-4f74-a38a-fd2b81fce50a)

**Comment by haruue** [comment]
https://hysteria.network/docs/Artworks/

**Reply by specked2527** [reply]
https://github.com/apernet/hysteria-hideri-stickers
https://hysteria.network/docs/Artworks/
这里没有看到说明形象设计和来源是什么？

![06](https://github.com/user-attachments/assets/a39b9c3d-7a03-4a80-92ee-c2572d15b3e8)

**Reply by haruue** [reply, marked as answer]
抱歉，我没能理解您问的「形象设计和来源」是指什么。

「彩羽ひでり」的设定是我们寻找合作画师根据我们提供的一些设定要求进行设计的， 未参考任何既存的角色。

如果您需要了解提供该设计的画师信息。 很抱歉根据我们当初和画师的达成共识， 在画师未明确提出要求的情况下， 我们不会披露以下两个作品的画师信息：
+ [主视觉图](https://hysteria.network/assets/artworks/hc_name.png)
+ 表情包（第一弹）

**Reply by specked2527** [reply]
角色形象设计的创作流程和灵感来源
> 「彩羽ひでり」的设定是我们寻找合作画师根据我们提供的一些设定要求进行设计的， 未参考任何既存的角色。

这就足够了，虽然我想知道其他细节，
 如；彩羽ひでり的背景故事，发卡和角有什么意义？

目前的理解就是；彩羽ひでり---来自hysteria Project吉祥物 
没有其他信息了，当然，没有也是可以的

**Reply by haruue** [reply]
龙角来自 Hysteria 的前作 —— dragonite
发卡以及「雷属性/電気属性」的角色设定来自于 Hysteria 2 图标。
